aten/src/ATen/native/mps/MPSGraphVenturaOps.h

```diff
@@ -3,18 +3,90 @@
+    MPSGraphResizeNearestRoundingModeCeil              =  2L,
+    MPSGraphResizeNearestRoundingModeFloor             =  3L,
+    MPSGraphResizeNearestRoundingModeRoundToEven       =  4L,
+    MPSGraphResizeNearestRoundingModeRoundToOdd        =  5L,
+};
+#endif
+
+- (MPSGraphTensor * _Nonnull)cumulativeSumWithTensor:(MPSGraphTensor * _Nonnull)tensor
                                        axis:(NSInteger)axis
-                                       name:(NSString *)name;
+                                       name:(NSString * _Nullable)name;
 
-- (MPSGraphTensor *)sortWithTensor:(MPSGraphTensor *)tensor
+- (MPSGraphTensor * _Nonnull)sortWithTensor:(MPSGraphTensor * _Nonnull)tensor
                                        axis:(NSInteger)axis
-                                       name:(NSString *)name;
+                                       name:(NSString * _Nullable)name;
 
-- (MPSGraphTensor *)argSortWithTensor:(MPSGraphTensor *)tensor
+- (MPSGraphTensor * _Nonnull)argSortWithTensor:(MPSGraphTensor * _Nonnull)tensor
                                        axis:(NSInteger)axis
-                                       name:(NSString *)name;
+                                       name:(NSString * _Nullable)name;
+
+- (MPSGraphTensor * _Nonnull)inverseOfTensor:(MPSGraphTensor * _Nonnull) inputTensor
+                                        name:(NSString * _Nullable)name;
+
+- (MPSGraphTensor * _Nonnull) resizeNearestWithTensor:(MPSGraphTensor * _Nonnull) imagesTensor
+                                  sizeTensor:(MPSGraphTensor * _Nonnull) size
+                         nearestRoundingMode:(MPSGraphResizeNearestRoundingMode) nearestRoundingMode
+                                centerResult:(BOOL) centerResult
+                                alignCorners:(BOOL) alignCorners
+                                      layout:(MPSGraphTensorNamedDataLayout) layout
+                                        name:(NSString * _Nullable) name;
+
+- (MPSGraphTensor * _Nonnull) resizeNearestWithTensor:(MPSGraphTensor * _Nonnull) imagesTensor
+                                  sizeTensor:(MPSGraphTensor * _Nonnull) size
+                           scaleOffsetTensor:(MPSGraphTensor * _Nonnull) scaleOffset
+                         nearestRoundingMode:(MPSGraphResizeNearestRoundingMode) nearestRoundingMode
+                                      layout:(MPSGraphTensorNamedDataLayout) layout
+                                        name:(NSString * _Nullable) name;
+
+- (MPSGraphTensor * _Nonnull) resizeBilinearWithTensor:(MPSGraphTensor * _Nonnull) imagesTensor
+                                   sizeTensor:(MPSGraphTensor * _Nonnull) size
+                                 centerResult:(BOOL) centerResult
+                                 alignCorners:(BOOL) alignCorners
+                                       layout:(MPSGraphTensorNamedDataLayout) layout
+                                         name:(NSString * _Nullable) name;
```

Comment:
Is this the rendering issue, or can you please align message parameters by colon?